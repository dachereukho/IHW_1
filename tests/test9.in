7
4
1
3
4
-9
9
6