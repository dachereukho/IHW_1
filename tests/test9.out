1 4 9 4 3 -9 6